utilities/spreadsheet_docx/copy_from_bailii_s3.py

```diff
@@ -0,0 +1,171 @@
+# ruff: noqa: T201 (print)
+# ruff: noqa: S603 (untrusted subprocess input)
+# ruff: noqa: PYI024 (typing.NamedTuple instead of collections)
+
+
+# """Make a bash script to copy the files across to S3"""
+
+import csv
+import json
+import re
+import subprocess
+import sys
+from collections import Counter, namedtuple
+from pathlib import Path
+
+import requests
+
+SOURCE_BUCKET = "bailii-docx"
+UNPUBLISHED_BUCKET = "tna-caselaw-unpublished-assets"
+PUBLISHED_BUCKET = "tna-caselaw-assets"
+DRY_RUN = "--dryrun"
+# DRY_RUN = False
+DALMATIAN_INFRASTRUCTURE = "caselaw"
+ASSETS_BASE = "https://tna-caselaw-assets.s3.amazonaws.com"
+CASELAW_BASE = "https://caselaw.nationalarchives.gov.uk"
+
+SPREADSHEET = "bailii_files.csv"
+
+# NOTE THIS MUST BE RUN AS A V2 DALMATIAN
+# Run `dalmatian version -v 2` first
+
+
+class UnexpectedExtension(Exception):
+    pass
+
+
+with Path.open(SPREADSHEET) as f:
```

Comment:
This raises the following with mypy:

```
No overload variant of "open" of "Path" matches argument type "str"
```

I think you want:

```suggestion
with Path(SPREADSHEET).open() as f:
```